src/layer/tile/GridLayer.js

```diff
@@ -446,11 +449,13 @@ L.GridLayer = L.Layer.extend({
 			this._tiles[key].current = false;
 		}
 
+		if (zoom < this.options.minZoom) { return; }
```

Comment:
I changed `_update` so it works on the gridlayer's "active zoom pyramid level" only, which is good. But if a tile layer has e.g. `minZoom=16` and the map is at zoom e.g. 2, then the code would generate a few million `<img>`s, then try to delete them later.

No check is needed for `maxZoom` because I really don't care about creating *fewer* tiles than normal, then deleting them.